tests/config/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 The clvk authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_gtest_executable(config_test
+    main.cpp
+    ${CLVK_PROJECT_SOURCE_DIR}/src/config.cpp
+    ${CLVK_PROJECT_SOURCE_DIR}/src/log.cpp
```

Comment:
Not sure if this is the right way to do this or not.

Reply:
I don't think we really want it that way.
I think we would want to do it like other tests using the `CLVK_UNIT_TESTING` meaning: include `unit.hpp` only, not `config.hpp` and use the function you have there. If you do not have everything add them in `unit.hpp`, but I think `clvk_get_config` should be enough for your purpose.